src/model/RacingGameModel.js

```diff
@@ -0,0 +1,61 @@
+import { INIT } from '../constants/constants.js';
+import CarModel from './CarModel.js';
+
+export default class RacingGameModel {
+  #cars = INIT.CARS;
+
+  #round = INIT.ROUND;
+
+  moveCars(moves) {
+    this.#cars = this.#cars.map((car, index) => {
+      return { ...car, position: car.position + moves[index] };
+    });
+  }
+
+  set cars(names) {
+    this.#cars = names.map((name) => new CarModel(name));
+  }
+
+  get cars() {
+    return this.#cars;
+  }
+
+  set round(newRound) {
+    this.#round = newRound;
+  }
+
+  get round() {
+    return this.#round;
+  }
+
+  get winners() {
+    return this.findWinners();
+  }
+
+  reset() {
+    this.#cars = INIT.CARS;
+    this.#round = INIT.ROUND;
+  }
+
+  findMaxRecord() {
+    const maxRecord = this.#cars.reduce(
+      (accumulator, current) => Math.max(accumulator, current.position),
+      INIT.RECORD
+    );
+
+    return maxRecord;
+  }
+
+  findWinners() {
+    const maxRecord = this.findMaxRecord();
+
+    const winnerList = this.#cars.reduce((accumulator, current) => {
+      if (current.position === maxRecord) {
+        accumulator.push(current.name);
+      }
+      return accumulator;
+    }, []);
```

Comment:
filter() 를 사용해서 한줄로 줄여볼까요?
```suggestion
    const winnerList = this.#cars.filter(...);
```

Reply:
네. 아래와 같이 수정했습니다. filter 메서드와 map 메서드를 체이닝하여 사용하니 무엇을 하려는 것인지 더 쉽게 읽히네요! 😄 

```javascript
    const winnerList = this.#cars
      .filter((car) => car.position === maxPosition)
      .map((car) => car.name);
```